api/streams_test.go

```diff
@@ -131,6 +132,40 @@ func TestStreamsOf_GetStream(t *testing.T) {
 			},
 			wantRcvd: 2,
 		},
+		{
+			name: "error: restarting stream",
+			fields: fields{
+				channels: map[string]*StreamChannelOf[*recordedClientStream, proto.Message]{
+					"mock:1234": {
+						dead: true,
+						channel: func() chan proto.Message {
+							// put 2 left over messages into the channel
+							var c = make(chan proto.Message)
+							go func() {
+								c <- &assessment.AssessEvidenceRequest{Evidence: &evidence.Evidence{Id: testdata.MockEvidenceID1}}
+							}()
+							go func() {
+								c <- &assessment.AssessEvidenceRequest{Evidence: &evidence.Evidence{Id: testdata.MockEvidenceID2}}
+							}()
+							return c
+						}(),
+						target:    "mock:1234",
+						component: "mock",
+					},
+				},
+			},
+			args: args{
+				target:    "mock:1234",
+				component: "mock component",
```

Comment:
the component name "mock component" in `args` is not the same as "mock" in `fields` but it doesn't matter here I guess

Reply:
It is still the same but the test passes